### PR TITLE
[ACR-1314] Remove deprecated APIs in guides

### PR DESCRIPTION
**Pull request by jiyong2**
- ACR: ACR-1314 (http://suprem.sec.samsung.net/jira/browse/ACR-1314)
Remove deprecated APIs in guides.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/482

File: docs/application/native/guides/multimedia/media-controller.md [hunk 1]
```diff
@@ -722,10 +642,6 @@ The following table lists all the server state attributes the client can receive
 | `MC_PLAYBACK_STATE_PLAYING`      | Playback state of playing                |
 | `MC_PLAYBACK_STATE_PAUSED`       | Playback state of paused                 |
 | `MC_PLAYBACK_STATE_STOPPED`      | Playback state of stopped                |
-| `MC_PLAYBACK_STATE_NEXT_FILE`    | Playback state of next file              |
-| `MC_PLAYBACK_STATE_PREV_FILE`    | Playback state of previous file          |
-| `MC_PLAYBACK_STATE_FAST_FORWARD` | Playback state of fast forward           |
-| `MC_PLAYBACK_STATE_REWIND`       | Playback state of rewind                 |
```
Comment (mijin85cho) [inline docs/application/native/guides/multimedia/media-controller.md]: Question. What is the difference between this `MC_PLAYBACK_STATE_REWIND` and `MC_PLAYBACK_STATE_REWIND` on line 732?

Comment (jiyong2) [reply docs/application/native/guides/multimedia/media-controller.md]: It is mistyping. So I changed 'MC_PLAYBACK_STATE_REWIND' to 'MC_PLAYBACK_STATE_REWINDING'. Thanks for your reviewing.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**